src/test/java/com/ft/methodearticlemapper/transformation/BodyProcessingFieldTransformerFactoryTest.java

```diff
@@ -1171,6 +1171,97 @@ public void shouldStripAllNotesFromContent() throws Exception {
         checkTransformation(contentWithNotes, transformedContent);
     }
 
+	@Test
+	public void shouldKeepAnchorTagsFromRecommended() throws Exception {
+		String contentWithNotes = "<body><recommended><recommended-title/><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		String transformedContent = "<body><recommended><recommended-title/><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		checkTransformation(contentWithNotes, transformedContent);
+	}
+
+	@Test
+	public void shouldRemoveEmptyIntroFromRecommended() throws Exception {
+		String contentWithNotes = "<body><recommended><p></p><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		String transformedContent = "<body><recommended><recommended-title/><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		checkTransformation(contentWithNotes, transformedContent);
+	}
+
+	@Test
+	public void shouldRemoveDummyTextIntroFromRecommended() throws Exception {
+		String contentWithNotes = "<body><recommended><p><?EM-dummyText [Title]?></p><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		String transformedContent = "<body><recommended><recommended-title/><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		checkTransformation(contentWithNotes, transformedContent);
+	}
+
+	@Test
+	public void shouldKeepIntroIfNotEmptyFromRecommended() throws Exception {
+		String contentWithNotes = "<body><recommended><p>Intro</p><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		String transformedContent = "<body><recommended><recommended-title/><p>Intro</p><ul><li><a href=\"link\">Title</a></li></ul></recommended></body>";
+		checkTransformation(contentWithNotes, transformedContent);
+	}
+
```

Comment:
Add a Test for the Headline from <li> tag. Check that if the Headline contains Dummy text it's ignored.